src/services/posts/fetch-posts.js

```diff
@@ -14,19 +14,21 @@ const fetchPosts = async(isFlagged) => {
 	 const fromClause = `FROM posts`
 	  
 	 const joinClause = `INNER JOIN users on posts."userId" = users.id`
-	 let whereClause = ''
+	 let whereClause = `WHERE posts."id" > ${cursor}` 
 	 let orderByClause = `ORDER BY posts."createdAt" DESC`
+	 const limitBy = `LIMIT ${limit}`
 	   
 	 if(isFlagged !== undefined){
 		 selectClause+= `, posts."flagsCount"`
 		 
-		whereClause += `
-			WHERE posts."flagsCount" ${isFlagged ? '>': '='} 0`
+		whereClause = ` ${whereClause} 
```

Comment:
@beblicarl please could you elaborate on why the change from `+=`

Reply:
oh it was a personal choice, but I will change it back